dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -160,7 +160,6 @@ public class ConnectorArguments extends DefaultArguments {
     private final OptionSpec<String> optionConfiguration = parser.accepts("config", "Configuration for DB connector").withRequiredArg().ofType(String.class).withValuesSeparatedBy(';').describedAs("key=val;key1=val1");
     // private final OptionSpec<String> optionDatabase = parser.accepts("database", "database (can be repeated; all if not specified)").withRequiredArg().describedAs("my_dbname").withValuesSeparatedBy(',');
     private final OptionSpec<File> optionOutput = parser.accepts("output", "Output file (cannot be used together with --output-dir)").withRequiredArg().ofType(File.class).describedAs("cw-dump.zip");
-    private final OptionSpec<String> optionOutputDir = parser.accepts("output-dir", "Output directory where the zip file with the default filename should be created (cannot be used together with --output option). If you want to specify the filename, use --output option instead.").withRequiredArg().ofType(String.class).defaultsTo("").describedAs("/home/user/my-dump-dir");
```

Comment:
Deleting the option is backwards incompatible change. Can we do that without the notice?
If we can, then I think it's fair to keep the old option, as a transitory measure / a grace period, but print a message and terminate the program if the option is supplied by the user. The message should cover the new ways of achieving the same thing (or explaining it's no longer supported).

Reply:
the flag was introduced in #138 2 days ago. I felt it's safe to remove for this reason.

Reply:
Not sure about the release process, but `output-dir` nevertheless was part of the https://github.com/google/dwh-migration-tools/releases/tag/v1.0.17

I'll allow that, since it's indeed a very fresh one, but I think in the future we must be careful in what changes we introduce (and revert!) to public API, especially if their logic is `isAssessment`-independent.